### PR TITLE
cleanup of project page

### PR DESCRIPTION
**Pull request by davidclaphan**
updates:
- works with multiple screen sizes
- navbar no longer crowding top row of projects 
- coloring is consistent with other app pages
- changed button to GitHub icon
- softer coloring on project shadows
- additional spacing between keywords for each project

### CODE REVIEW

File: frontend/src/components/Card.tsx [hunk 1]
```diff
@@ -23,11 +24,11 @@ export const ProjectCard = ( project: ProjectCardProps) => {
                     {project.projectDescription}
                 </div>
                 <div className='card-content-button'>
```
Comment (arse0039) [inline frontend/src/components/Card.tsx:26]: This class needs a CSS rule so that cursor changes when you mouse-over the icon.

**Review by arse0039 — CHANGES_REQUESTED** [review]
The page is a bit busy on larger screens.  If we add a max width to the layout, that could help
image
.  I like it when it allows for 4 cards max per row. 

The background color of the project page itself does not match the rest of the site. It is still using the dark gray/black instead of true black. 
We might want to add a max width for the individual cards as well because some browser widths look a bit thick:
image

**Comment by davidclaphan** [comment]
> The page is a bit busy on larger screens. If we add a max width to the layout, that could help. I like it when it allows for 4 cards max per row.
_**Updated so there is more spacing between cards and additional padding. On a fairly large screen it stops at 5, but still shows 3 wide on a smaller laptop screen**_


> The background color of the project page itself does not match the rest of the site. It is still using the dark gray/black instead of true black. 
**_updated this_**

> We might want to add a max width for the individual cards as well because some browser widths look a bit thick: 
**_fixed this one so the cards should stay the same size as large/small as you can make the browser_**

File: frontend/src/components/Card.tsx (hunk 1 above)
Comment (davidclaphan) [reply frontend/src/components/Card.tsx:26]: updated to simple pointer

**Review by arse0039 — APPROVED** [review]
Much better!

**Pull request merged**